docs/howto/contents.md

```diff
@@ -1,15 +1,27 @@
 # Azure Service Operator Documentation
 
-## Getting Started
+## Using Azure Service Operator
 
-1. [Prerequisites](prereqs.md):
-    Find the list of tools and dependencies needed to develop and deploy the Azure Service Operator here.
+1. [Build & deploy](deploy.md)
```

Comment:
I was a bit confused here, because the "build and deploy" and the "deploy with helm" look like two steps, which I should do both of.

I don't think that's actually the intent though, right?

Possibly consider changing this from a numbered list to a bulleted list (to emphasize that it's a "collection of unrelated resources" rather than an ordered set of instructions? Or maybe group the two different ways to install onto a single item in the list and clarify that you should pick one or the other (and we recommend... helm?)

Reply:
Possibly changing the title here from "Using Azure Service Operator" to "useful links" or "related topics" or something like that might also help, unsure.